targets/GenApi.targets

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Long-term, we should just be using the logic included in the GenAPI package's targets/props, right? So this file will be mostly unnecessary?